source/_integrations/fitbit.markdown

```diff
@@ -13,13 +13,43 @@ ha_integration_type: integration
 
 The Fitbit sensor allows you to expose data from [Fitbit](https://fitbit.com/) to Home Assistant.
 
+## Requirements before installation
+
 Prior to enabling the sensor, your Home Assistant instance must be enabled with SSL, using services such as [Home Assistant cloud](/cloud/) or [Duck DNS](/integrations/duckdns/) integrating Let's Encrypt. Fitbit will not allow insecure integrations.
 
+It is necessary to have configured the internal and external URLs. Settings-> System-> Network → (Home Assistant URL)
```

Comment:
It may be redundant and does not correspond to this integration, but the point is that it is not specified anywhere that this has to be complete.

Before setting up my HA Fitbit, everything was local, and I configured Duck DNS to integrate Fitbit, but during the process, I couldn't find a place where that configuration was defined.